system/I18n/Time.php

```diff
@@ -1005,7 +1005,7 @@ public function isAfter($testTime, ?string $timezone = null): bool
      */
     public function humanize()
     {
-        $now  = IntlCalendar::fromDateTime(self::now($this->timezone)->toDateTimeString());
+        $now  = IntlCalendar::fromDateTime(self::now($this->timezone));
```

Comment:
That's kind of funny that we were using an unnecessary intermediate string to begin with 😅